pkg/user/appointee/can_call.go

```diff
@@ -66,8 +59,9 @@ func canCall(cliCtx *cli.Context, generator func(logging.Logger, *canCallConfig)
 	}
 
 	result, err := elReader.UserCanCall(ctx, config.UserAddress, config.CallerAddress, config.Target, config.Selector)
-	logger.Infof("CanCall Result: %v", result)
-	logger.Infof("Selector, Target and User: %s, %x, %s", config.Target, string(config.Selector[:]), config.UserAddress)
+	fmt.Printf("CanCall Result: %v", result)
```

Comment:
you need a `\n` for new line at the end.

Reply:
Had seen other places explicitly using println for new lines -- I'll update.